### ISSUE TITLE
Hybrid mode doesn't support ng-content inside *ngIf

### ISSUE THREAD

**Issue by prasikumbhare** [angular/angular#9649]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
In Hybrid  mode (when using upgradeAdapter.bootstrap()), `<ng-content>` doesn't work if placed inside an `*ngIF`

**Expected/desired behavior**
it should be possible to include the light DOM children conditionally using *ngIF

**Reproduction of the problem**
Here is the plunk : [https://plnkr.co/edit/LaNrcc?p=preview](https://plnkr.co/edit/LaNrcc?p=preview)

**What is the expected behavior?**
it should be possible to include the light DOM children conditionally using *ngIF

**What is the motivation / use case for changing the behavior?**
Due to this bug, it's not possible to include the light DOM children conditionally.

**Please tell us about your environment:**

* **Angular version:** 2.0.0-rc.1
 
* **Browser:** [Chrome XX ]
 Tested in Chrome Version 51.0.2704.103 m
* **Language:** [all | TypeScript X.X | ES6/7 | ES5 | Dart] 
https://code.angularjs.org/tools/typescript.js

**Comment by youdz** [comment]
Duplicate of #6629.
#9538 fixes this.

**Comment by pkozlowski-opensource** [comment]
Closing as a duplicate of #6629.

**Comment by jensbodal** [comment]
For those like me that stumble across here, this is included in `4.0.0` but not `2.4.9`.

http://plnkr.co/edit/6SFy7nbK6TgtfQ44tMfs?p=preview

Change `ngVer` in `systemjs.config.js` to `4.0.0` to see it work.
